recipe/migrations/arrow_cpp500.yaml

```
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+arrow_cpp:
+- 5.0.0
+migrator_ts: 1627598979.4913893
```

Maybe something like this instead?

```suggestion
__migrator:
  build_number: 1
  kind: version
  migration_number: 1
  operation: key_add
  primary_key: arrow_cpp
arrow_cpp:
- 3.0.0
- 4.0.1
- 5.0.0
migrator_ts: 1627598979.4913893
```

---

Though we might not need the keyadd migrator, i think this is a simple enough one